storage/aptosdb/src/pruner/transaction_store/test.rs

```diff
@@ -60,7 +118,7 @@ fn verify_txn_store_pruner(
         &txns,
     );
 
-    // start pruning transactions batches of size 2 and verify transactions have been pruned from DB
+    // start pruning transactions batches of size step_size and verify transactions have been pruned from DB
```

Comment:
nit: not introduced by this diff, but why is `step_size` an `i32`?

Reply:
That's an oversight - let me fix it.